dbtmetabase/__init__.py

```diff
@@ -227,23 +278,41 @@ def main(args: List = None):
         logger.addHandler(logging.StreamHandler(sys.stdout))
         logger.setLevel(logging.DEBUG)
 
+    # These args drive loading the Metabase client and dbt models and are prerequisites to any functionality of dbt-metabase
+    primary_args = {
+        "dbt_path": parsed.dbt_path,
+        "dbt_manifest_path": parsed.dbt_manifest_path,
+        "dbt_database": parsed.dbt_database,
+        "metabase_host": parsed.metabase_host,
+        "metabase_user": parsed.metabase_user,
+        "metabase_password": parsed.metabase_password,
+        "metabase_use_http": parsed.metabase_use_http,
+        "metabase_verify": parsed.metabase_verify,
+        "metabase_database": parsed.metabase_database,
+        "schema_excludes": parsed.schema_excludes,
+        "metabase_sync_skip": parsed.metabase_sync_skip,
+        "metabase_sync_timeout": parsed.metabase_sync_timeout,
+        "includes": parsed.includes,
+        "excludes": parsed.excludes,
+    }
+
     if parsed.command == "export":
-        export(
-            dbt_database=parsed.dbt_database,
-            dbt_manifest_path=parsed.dbt_manifest_path,
-            dbt_path=parsed.dbt_path,
-            dbt_docs_url=parsed.dbt_docs,
-            metabase_database=parsed.metabase_database,
-            metabase_host=parsed.metabase_host,
-            metabase_user=parsed.metabase_user,
-            metabase_password=parsed.metabase_password,
-            metabase_use_http=parsed.metabase_use_http,
-            metabase_verify=parsed.metabase_verify,
-            metabase_sync_skip=parsed.metabase_sync_skip,
-            metabase_sync_timeout=parsed.metabase_sync_timeout,
-            schema=parsed.schema,
-            schema_excludes=parsed.schema_excludes,
-            includes=parsed.includes,
-            excludes=parsed.excludes,
+        execute(
+            **primary_args,
             include_tags=parsed.include_tags,
+            dbt_docs_url=parsed.dbt_docs_url,
+            command="export_models",
+        )
+    elif parsed.command == "exposures":
+        execute(
+            **primary_args,
+            output_path=parsed.output_path,
+            output_name=parsed.output_name,
+            include_personal_collections=parsed.include_personal_collections,
+            exclude_collections=parsed.exclude_collections,
+            command="extract_exposures",
+        )
+    else:
+        logging.warning(
+            "Invalid command. Must be one of either 'export' or 'exposures'."
```

Comment:
Would you say it "exports" the exposures as well or am I misunderstanding what this is doing? If so, maybe we have `export-models` and `export-exposures` or even `models` and `exposures`?

Reply:
Hm well since we called export being to movement of information from dbt -> metabase,
this is essentially the inverse; so if speaking from the perspective of the dbt project-
We would export model defs to metabase
We would import metabase exposures to dbt project

But to save for unneeded complexity, I like the idea of just `models` and `exposures` (and eventually `metrics` to round everything out). Very simple that way which is good for users.